managed-ledger/src/main/java/org/apache/bookkeeper/mledger/util/CallbackMutex.java

```diff
@@ -61,9 +61,8 @@ public boolean tryLock() {
     public void unlock() {
         if (log.isDebugEnabled()) {
             owner = null;
-            position = null;
-            log.debug(">>> Lock {} released token={} at {}", this.hashCode(),
-                    Thread.currentThread().getStackTrace()[2]);
+            position = Thread.currentThread().getStackTrace()[2].toString();
```

Comment:
Can `Thread.currentThread().getStackTrace()[2]` return `null`?

Reply:
1. This method Thread.currentThread().getStackTrace() will not return null.

2. StackTraceElement[] array = Thread.currentThread().getStackTrace();
	
	Usually, the length of array is greater than or equal to 2 when the thread is running.

	I did a test.

	The length is 2,when this method was called in Main(),otherwise the length is greater than 2.

3. Also, I ran the test case(unlock) in org.apache.bookkeeper.mledger.util.CallbackMutexTest.java.
	The test results were normal.
	
4. So, I think calling here, Thread.currentThread().getStackTrace()[2]  will not return null.